src/screens/AirdropFinishedWalletDetails/AirdropFinishedWalletDetails.tsx

```diff
@@ -16,7 +16,9 @@ export const AirdropFinishedWalletDetails: FC<Props> = ({ route }: Props) => (
   <ScreenTemplate header={<Header isBackArrow title={i18n.airdrop.title} />}>
     <Text style={styles.subtitle}>{i18n.airdrop.finished.subtitle}</Text>
     <AirdropWalletBalanceCard walletDetails={{ balance: route.params.balance, name: route.params.name }} />
-    <AirdropStayTuned />
+    <View style={styles.stayTunedContainer}>
+      <AirdropStayTuned />
```

Comment:
I'm wondering if it wouldn't be better to pass `styles.stayTunedContainer` in the properties to `AirdropStayTuned` and remove this `View` wrapper here. What do you think?

Reply:
I think container is better, it s not AirdropStayTuned component business what s the margin in this case. It differs in every view so I would rather have it defined in the view